src/fable/Fable.Client.Node/js/fable.js

```diff
@@ -386,8 +386,9 @@ function readFableConfigOptions(opts) {
         var projDir = fs && fs.statSync(fullProjFile).isDirectory()
                         ? fullProjFile
                         : path.dirname(fullProjFile);
-        cfgFile = fableLib.pathJoin(projDir, constants.FABLE_CONFIG_FILE);
-
+        if (fs && !fs.existsSync(cfgFile)) {
+            cfgFile = fableLib.pathJoin(projDir, constants.FABLE_CONFIG_FILE);
+        }
```

Comment:
Not sure this is a good idea. For example in [fable-powerpack](https://github.com/fable-compiler/fable-powerpack) there's a fableconfig file in the root dir and in the tests dir. If I call (from root) `fable tests` I expect the fableconfig file in `tests` directory to be read, not the one in the current (root) dir. That will fail with this change, can you please revert it?